server/src/metadata/tenant-migration/tenant-migration.entity.ts

```diff
@@ -5,29 +5,37 @@ import {
   PrimaryGeneratedColumn,
 } from 'typeorm';
 
-export type TenantMigrationColumnChange = {
+export type TenantMigrationColumnAction = {
   name: string;
   type: string;
-  change: 'create' | 'alter';
+  action: 'create';
 };
 
-export type TenantMigrationTableChange = {
+export type TenantMigrationTableAction = {
   name: string;
-  change: 'create' | 'alter';
-  columns?: TenantMigrationColumnChange[];
+  action: 'create' | 'alter';
+  columns?: TenantMigrationColumnAction[];
 };
-
 @Entity('tenant_migrations')
 export class TenantMigration {
   @PrimaryGeneratedColumn('uuid')
   id: string;
 
   @Column({ nullable: true, type: 'jsonb' })
-  migrations: TenantMigrationTableChange[];
+  migrations: TenantMigrationTableAction[];
+
+  @Column({ nullable: true })
+  name: string;
+
+  @Column({ default: false })
+  isCustom: boolean;
```

Comment:
It sounds redundant since we should not have anything inside of "name" for custom migrations, but it's not too costly either to have it and it's clearer to fetch custom migrations using that flag than a rule "name IS NULL"